docs/manual_configuration.md

```diff
@@ -0,0 +1,201 @@
+## Manual Installation
+Follow these steps if you choose to forgo the generator:
+
```

Comment:
You might want to build a small sample app using these steps, or see the pull requests here showing what the various options do.